src/models/UserMapper.php

```diff
@@ -642,4 +642,14 @@ public function resetPassword($token, $password)
 
         return false;
     }
+
+
+    public function delete($user_id)
+    {
+        $sql  = "delete from user where ID = :user_id";
+        $stmt = $this->_db->prepare($sql);
+        $stmt->execute(array("user_id" => $user_id));
+
+        return true;
```

Comment:
How can this always return ```true```? Shouldn't we check the result of the transaction and decide on that what to return?